tox.ini

```diff
@@ -5,40 +5,46 @@ envlist =
     flake8,
     isort,
     py35-2.2,
-    py36-{2.2,3.0,master},
-    py37-{2.2,3.0,master},
-    py38-{2.2,3.0,master},
+    py36-{2.2,3.2,main},
+    py37-{2.2,3.2,4.0,main},
+    py38-{2.2,3.2,4.0,main},
+    py39-{2.2,3.2,4.0,main},
 
 [tox:travis]
 3.5 = py35
 3.6 = py36
 3.7 = py37
 3.8 = py38
+3.9 = py39
 
 [testenv]
 basepython =
     py35: python3.5
     py36: python3.6
     py37: python3.7
     py38: python3.8
+    py39: python3.9
 usedevelop = true
 commands =
-    {envpython} -R -Wonce {envbindir}/coverage run --branch -m django test -v2 --settings=tests.settings {posargs}
+    {envpython} -R -Wonce {envbindir}/coverage run --branch -m pytest --ds tests.settings tests/tests.py {posargs}
```

Comment:
What's the purpose of switching to `pytest` and requiring two now dependencies? I think the default Django test runner should be fine for this project.